script/results.html

```diff
@@ -613,10 +638,18 @@
             for (const item of input[key]) {
               const duration = dayjs.duration(item.duration / 1000000)
               formattedDuration = duration.format('D[d] H[h] m[m] s[s] SSS[ms]')
+              skippedReason = ""
+              if (item.state == "skipped") {
+                skippedReason = item.failureReason
+                if (skippedReason == "") {
+                  skippedReason = "Test case skipped by user"
```

Comment:
```suggestion
                  skippedReason = "Test case skipped by configuration"
```

Minor: Maybe change this to "configuration" instead of placing blame on the user themselves.

Reply:
fixed